app/dashboard/utils.py

```diff
@@ -0,0 +1,194 @@
+    standardbounties_addr = getStandardBountiesContractAddresss(network)
+    bounty_abi = json.loads(standardbounties_abi)
+    getBountyContract = web3.eth.contract(standardbounties_addr, abi=bounty_abi)
+    return getBountyContract
+
+
+def get_bounty(bounty_enum, network):
+    try:
+        ipfs = getIPFS()
+        standard_bounties = getBountyContract(network)
+        issuer, deadline, fulfillmentAmount, paysTokens, bountyStage, balance = standard_bounties.functions.getBounty(bounty_enum).call()
+        bountydata = standard_bounties.functions.getBountyData(bounty_enum).call()
+        arbiter = standard_bounties.functions.getBountyArbiter(bounty_enum).call()
+        token = standard_bounties.functions.getBountyToken(bounty_enum).call()
+        fulfillments = []
+        fulfill_enum = 0
+        more_fulfillments = True
+        while more_fulfillments:
+            try:
+                accepted, fulfiller, data = standard_bounties.functions.getFulfillment(bounty_enum, fulfill_enum).call()
```

Comment:
Instead of just looping through the fulfillments until there is an error, could we use `getNumFulfillments()` first?  And then just iterate N times?

Reply:
good idea.. updated